src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swixproj

```diff
@@ -6,15 +6,16 @@
   <PropertyGroup>
     <OutputArchitecture>neutral</OutputArchitecture>
     <OutputLocalized>false</OutputLocalized>
-    <OutputPath>$(OutDir)Insertion</OutputPath>
+    <RoslynOutputPath>$(OutputPath)</RoslynOutputPath>
+    <OutputPath>$(OutputPath)\Insertion</OutputPath>
```

Comment:
This now being dependent upon itself is somewhat terrifying to me in that it's now more sensitive to ordering than it was before.